.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5502
+    "liveServer.settings.port": 5503
```

Comment:
[OPITONAL] Kindly remove `.vscode` from your file. To make your repository clean.